runtime/src/main/scala/com/ing/baker/runtime/core/BakerResponse.scala

```diff
@@ -62,8 +69,25 @@ class BakerResponse(processId: String, source: Source[TransitionResponse, NotUse
 
   val (receivedFuture, completedFuture) = BakerResponse.createFlow(processId, source)
 
-  def confirmReceived(implicit timeout: FiniteDuration): Unit = Await.result(receivedFuture, timeout)
-
-  def confirmCompleted(implicit timeout: FiniteDuration): Unit = Await.result(completedFuture, timeout)
+  def confirmReceived(implicit timeout: FiniteDuration): Boolean = {
```

Comment:
Is returning a boolean the best solution here? Alternatively we can throw an exception, or never throw an exception and have some kind of response enumeration: ACCEPTED, REJECTED, ERROR